workshops/templates/workshops/all_profileupdaterequests.html

```diff
@@ -17,7 +17,7 @@
     <tbody>
     {% for req in requests %}
       <tr>
-        <td>{{ req.personal }} {{ req.family }}</td>
+        <td>{{ req.personal }} {{ req.middle }} {{ req.family }}</td>
```

Comment:
Oh sorry, my mistake. This and https://github.com/swcarpentry/amy/pull/1061/commits/015b2c26063ce4433b4785e6876aa02797b81f56#diff-e4b786fea3b24ff03fcc3d341a1c213bR63 are actually for different models, so there's no real benefit from doing `person_name` property for each model. My bad.